Src/Microsoft.Dynamic/Interpreter/Instructions/LessThanInstruction.cs

```diff
@@ -8,118 +8,238 @@
 using Microsoft.Scripting.Utils;
 
 namespace Microsoft.Scripting.Interpreter {
-    public abstract class LessThanInstruction : Instruction {
+    internal abstract class LessThanInstruction : Instruction {
+        public static readonly object BoxedFalse = false;
+        public static readonly object BoxedTrue = true;
```

Comment:
Where should we place the above two lines? In a new class somewhere?

Reply:
There are ScriptingRuntimeHelpers.True/False which we should use instead.

Reply:
@gpetrou Are you still wanting this merged? Can you update to use the ScriptingRuntimeHelpers?

Reply:
@slide I did as you asked. Do you want to merge this?